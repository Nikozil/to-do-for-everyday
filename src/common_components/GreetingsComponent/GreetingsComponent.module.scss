.greeting-page {
  display: flex;
  min-height: 100vh;
  justify-content: center;
  background-color: #354052;

  &__content {
    min-width: 200px;
    max-width: 400px;
    width: 100%;
    margin: 0 5px 0 5px;
    color: #fff;
  }

  &__logo {
    margin-top: 200px;

    display: flex;
    justify-content: center;

    @media screen and (max-width: 767px) {
      margin-top: 100px;
    }
  }
}
