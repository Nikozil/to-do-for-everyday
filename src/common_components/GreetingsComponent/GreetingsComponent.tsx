import React from 'react';
import { Link } from 'react-router-dom';
import LogoComponent from '../LogoComponent/LogoComponent';
import styles from './GreetingsComponent.module.scss';

const GreetingsComponent: React.FC = ({ children }) => {
  return (
    <div className={styles['greeting-page']}>
      <div className={styles['greeting-page__content']}>
        <Link to="/">
          <LogoComponent className={styles['greeting-page__logo']} />
        </Link>
        {children}
      </div>
    </div>
  );
};

export default GreetingsComponent;
