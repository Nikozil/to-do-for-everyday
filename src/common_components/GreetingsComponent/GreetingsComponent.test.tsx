import { render } from '../../utils/tests/test-utils';
import GreetingsComponent from './GreetingsComponent';

describe('GreetingsComponent tests', () => {
  it('render correct', () => {
    const { asFragment } = render(<GreetingsComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
