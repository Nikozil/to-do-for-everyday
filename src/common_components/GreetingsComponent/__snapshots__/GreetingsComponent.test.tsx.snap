// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GreetingsComponent tests render correct 1`] = `
<DocumentFragment>
  <div
    class="greeting-page"
  >
    <div
      class="greeting-page__content"
    >
      <a
        href="/"
      >
        <div
          class="greeting-page__logo"
        >
          <img
            alt="Logo"
            src="Logo.png"
          />
        </div>
      </a>
    </div>
  </div>
   
</DocumentFragment>
`;
